js/homeview.js

```diff
@@ -258,7 +258,10 @@ enyo.kind({
 				base_angle = PI2/activitiesCount;
 			} else {
 				restrictedMode = true; spiralMode = false;
-				activitiesCount = parseInt(circumference/icon_padding)-1;
+				activitiesCount = parseInt(circumference/icon_padding);
+				while((circumference/activitiesCount) <= 1.18*constant.iconSpacingFactor*icon_padding){    
```

Comment:
I suggest to replace the literal `1.18` by a constant.
You could reuse `constant.ringSpaceFactor` in `constant.js`. It's no longer use anywhere.